### PR TITLE
Coins

### PR DESCRIPTION
**Pull request by ScottyXIII**

### CODE REVIEW

File: src/objects/entities/Ben3.ts [hunk 1]
```diff
@@ -5,12 +5,25 @@ import matterAddImageEllipse from '@/helpers/matterAddImageEllipse';
...
+import useLocalStorage from '@/helpers/useLocalStorage';
 
 const KEY = 'ben3';
 
 const HEAD_SCALE_MIN = 0.1;
 const HEAD_SCALE_MAX = 0.5;
 
+// @ts-expect-error todo
+// Can't find correct type here? tried settting data to MatterJS.ICollisionPair but gameobject don't exsit on it.
+const onCollision = data => {
```
Comment (ScottyXIII) [inline src/objects/entities/Ben3.ts:17]: Can't find correct type here? did try MatterJS.ICollisionPair but gameobject don't exist on it. I also tried 
` MatterJS.ICollisionPair & Entity`
` MatterJS.ICollisionPair & {gameobject: Entity}`

File: public/level/tiled-level/test-bumpy.json [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+         "spacing":0,
+         "tilecount":45,
+         "tileheight":32,
+         "tilewidth":32
+        }],
+ "tilewidth":32,
+ "type":"map",
+ "version":"1.10",
+ "width":30
+}
```
Comment (norgeous) [inline public/level/tiled-level/test-bumpy.json:327]: i guess we should delete this file?

File: public/level/tiled-level/test-64c.json [hunk 3]
```diff
@@ -1,2424 +1,2422 @@
...
+                        {
+                         "x":0,
+                         "y":-512
+                        }, 
+                        {
+                         "x":2880,
+                         "y":-512
+                        }, 
+                        {
+
```
Comment (norgeous) [inline public/level/tiled-level/test-64c.json:2422]: this file needs to be linted

File: public/level/tiled-level/test-bumpy.json (hunk 2 above)
Comment (ScottyXIII) [reply public/level/tiled-level/test-bumpy.json:327]: hmmm thought that would have been removed when I merged main 🤔 it's gone now!

File: public/level/tiled-level/test-64c.json (hunk 3 above)
Comment (ScottyXIII) [reply public/level/tiled-level/test-64c.json]: done!

File: src/objects/entities/Ben3.ts (hunk 1 above)
Comment (norgeous) [reply src/objects/entities/Ben3.ts:17]: it does work with:
```
type MyCollisionPair = MatterJS.ICollisionPair & {
  bodyB: { gameObject: Entity };
};
```
you were so close... :+1: 

then just need to update Entity.ts class var to public
```
public collisionCategory: CollisionCategories | undefined;
```

File: public/level/tiled-level/test-bumpy.json (hunk 2 above)
Comment (norgeous) [reply public/level/tiled-level/test-bumpy.json:327]: its still here :thinking:

File: src/objects/entities/Ben3.ts (hunk 1 above)
Comment (ScottyXIII) [reply src/objects/entities/Ben3.ts]: Oooooh dam! I missed the BodyB param as well.... yeah that makes sense. Nice one 👍

**Review by norgeous — APPROVED** [review]

File: public/level/tiled-level/test-bumpy.json (hunk 2 above)
Comment (ScottyXIII) [reply public/level/tiled-level/test-bumpy.json]: ok should be gone now, I didn't commit the deleted file!

**Pull request merged**